### PR TITLE
Commercial Invoice updates

### PR DESCRIPTION
**Pull request by nissimsan**
This PR: 
- Introduces line items (instead of listing products)
- Adds missing parties

### CODE REVIEW

File: docs/openapi/components/schemas/common/Product.yml [hunk 1]
```diff
@@ -11,6 +11,13 @@ properties:
       - type: string
         enum:
           - Product
+  productID:
```
Comment (mkhraisha) [inline docs/openapi/components/schemas/common/Product.yml:14]: I've wanted this for so long!!!

**Review by mkhraisha — APPROVED** [review]
FYI this PR does a lot more than just CI fixes.. it changes a lot of very commonly used schemas.
I would appreciate it if you update the PR title, and description, so other reviewers give it the attention it deserves. For example this updates Commercial invoice and Product schemas with new fields, 

Generally happy with it though

File: docs/openapi/components/schemas/common/CommercialInvoice.yml [hunk 2]
```diff
@@ -18,50 +18,98 @@ properties:
...
     description: Port where the purchased goods will enter first.
-    $ref: ./PostalAddress.yml
+    $ref: ./Place.yml
     $linkedData:
       term: portOfEntry
-      '@id': https://schema.org/address
+      '@id': https://schema.org/Place
+  originCountry:
+    title: Origin Country
+    description: A country of origin for the consignment, consignment item or product.
```
Comment (TallTed) [inline docs/openapi/components/schemas/common/CommercialInvoice.yml:45]:
```suggestion
    description: A country of origin for the consignment, consignment item, or product.
```

File: docs/openapi/components/schemas/common/CommercialInvoice.yml [hunk 3]
```diff
@@ -18,50 +18,98 @@ properties:
...
+  destinationCountry:
+    title: Destination Country
+    description: Country to which the purchased product will be delivered
+    type: string
+    $linkedData:
+      term: destinationCountry
+      '@id': https://service.unece.org/trade/uncefact/vocabulary/uncefact/#destinationCountry
+  invoiceDate:
+    title: Invoice Date
+    description: A date, time, date time or other date time value of the invoice in the trade settlement.
```
Comment (TallTed) [inline docs/openapi/components/schemas/common/CommercialInvoice.yml:59]:
```suggestion
    description: A date, time, date time, or other date time value of the invoice in the trade settlement.
```

File: docs/openapi/components/schemas/common/CommercialInvoice.yml [hunk 4]
```diff
@@ -18,50 +18,98 @@ properties:
...
     $linkedData:
       term: purchaseDate
       '@id': https://schema.org/paymentDueDate
-  totalPaymentDue:
-    title: Total Payment Due
-    description: The total amount due.
-    $ref: ./PriceSpecification.yml
+  seller: 
+    title: Seller
+    description: An entity which offers (sells / leases / lends / loans) the services / goods. A seller may also be a provider.
```
Comment (TallTed) [inline docs/openapi/components/schemas/common/CommercialInvoice.yml:73]:
```suggestion
    description: An entity which offers (sells, leases, lends, or loans) the services or goods. A seller may also be a provider.
```

File: docs/openapi/components/schemas/common/CommercialInvoice.yml [hunk 5]
```diff
@@ -72,23 +120,89 @@ properties:
...
+  termsOfDelivery: 
+    title: Terms of Delivery
+    description: The conditions agreed upon between the parties with regard to the delivery of goods and or services.
+    type: string
+    $linkedData:
+      term: termsOfDelivery
+      '@id': https://service.unece.org/trade/uncefact/vocabulary/uncefact/#specifiedDeliveryTerms
+  termsOfPayment: 
+    title: Terms of Payment
+    description: Terms, conditions, and currency of settlement as agreed upon by the vendor and purchaser per the pro forma invoice, customer purchase order and/or the letter of credit.
```
Comment (TallTed) [inline docs/openapi/components/schemas/common/CommercialInvoice.yml:162]:
```suggestion
    description: Terms, conditions, and currency of settlement, as agreed upon by the vendor and purchaser per the pro forma invoice, customer purchase order, and/or the letter of credit.
```

File: docs/openapi/components/schemas/common/Package.yml [hunk 6]
```diff
@@ -24,32 +24,20 @@ properties:
...
-    title: Included Trade Line Item
-    description: An included trade line item.
-    type: array
-    items:
-      $ref: ./PartOfOrder.yml
+      term: quantity
+      '@id': https://www.gs1.org/voc/netContent
+  packagingType:
+    title: Packaging Type
+    description: Links to details about packaging material type, quantity and thickness
```
Comment (TallTed) [inline docs/openapi/components/schemas/common/Package.yml:36]:
```suggestion
    description: Links to details about packaging material type, quantity, and thickness
```

**Review by OR13 — APPROVED** [review]

**Review by BenjaminMoe — APPROVED** [review]

**Review by mprorock — APPROVED** [review]

**Pull request merged**